internal/super/flix.go

```diff
@@ -234,9 +316,56 @@ func getTemplate(state *flowkit.State, flixService flixkit.FlixService, flixQuer
 		if err != nil {
 			return nil, fmt.Errorf("could not parse flix from file %s: %w", flixQuery, err)
 		}
+	case flixUrl:
+		flixString, err := flixkit.FetchFlixWithContext(ctx, flixQuery)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse flix from url %s: %w", flixQuery, err)
+		}
+		template, err = flixkit.ParseFlix(flixString)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse flix from url %s: %w", flixQuery, err)
+		}
 
 	default:
 		return nil, fmt.Errorf("invalid flix query type: %s", flixQuery)
 	}
 	return template, nil
 }
+
+// GetRelativePath computes the relative path from target to source.
+func GetRelativePath(source, target string) (string, error) {
+	relPath, err := filepath.Rel(filepath.Dir(target), source)
```

Comment:
this is backwards https://pkg.go.dev/path/filepath#Rel.

Reply:
noice, I'll update. thanks

Reply:
It's correct. I'll update the method params so it's clear what is going on.